### PR TITLE
Closes #17 - clean-up the code

### LINKED ISSUE #17: Clean up
Todos:
  - Separation of .md files vs other files
  - Folder structure same as the website structure
  - Static folder for docs - the structure of the content folders should be replicated 
  - File naming lower camelCase spi not capitalized
  - File names same as item name on website
  - Customize features category file
  - Check all category files and rework if needed

### PR DESCRIPTION
**Pull request by ryzheboka**
Does the following:
    - Separation of .md files vs other files
    - Deletion of unneeded imports
    - Folder structure same as the website structure
    - Static folder for docs - the structure of the content folders should be replicated
    - File names same as item name on website
    - Rework features category file
    - Check all other category files and rework if needed

### CODE REVIEW

File: docs/user-guide/configuration/taskana-jobs.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+| taskana.jobs.cleanup.history.simple.batchSize     | upper bound of how many history events can be processed by one history cleanup job | 45           |
+
+## TaskUpdatePriorityJob Configuration
+
+| Parameter                        | Description                                                           | Sample Value         |
+|----------------------------------|-----------------------------------------------------------------------|----------------------|
+| taskana.jobs.priority.task.batchSize   | upper bound of how many tasks can be processed by one TaskUpdatePriorityJob                | 70                  |
+| taskana.jobs.priority.task.runEvery   | period of time between the executions of the TaskUpdatePriorityJob (Duration in ISO 8601 format)   | P2D                  |
+| taskana.jobs.priority.task.firstRunAt | first time the job is executed (DateTime in ISO 8601 format)                 | 2021-08-03T08:00:00Z |
+| taskana.jobs.priority.task.active     | the job will only be executed if the flag is set to true | true                |
```
Comment (gitgoodjhe) [inline docs/user-guide/configuration/taskana-jobs.md:67]: isn't this taskana.jobs.priority.task.enable? Same for the other jobs
Also, if this is set to false, it won't even be scheduled, which technically is a difference to not being executed (but scheduled and therefore having an entry in the database)

File: docs/user-guide/configuration/taskana-jobs.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+- 5th April 6pm: The Job becomes due. However, it isn't run yet.
+- 5th April 7pm: Scheduler runs the first time. It runs the Job.
+- 5th April 9pm: The Job becomes due. However, it isn't run yet.
+- 5th Aril 10pm: Scheduler runs the second time. It runs the Job.
+- 5th Aril 12pm: The Job becomes due. However, it isn't run yet.
+- 6th April 1am: Scheduler runs the third time. It runs the Job.
+- 6th Aril 3am: The Job becomes due. However, it isn't run yet.
+
+
+## TaskCleanupJob and HistoryCleanupJob Configuration
```
Comment (gitgoodjhe) [inline docs/user-guide/configuration/taskana-jobs.md:48]: For all jobs: Could you change the "Sample Value" to our actual default values? This gives a sample as well and gives even more info with the actual default. Also name the column default value pls

File: docs/user-guide/configuration/taskana-jobs.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+| Parameter                               | Description                                                                                                               | Sample Value         |
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------|----------------------|
+| taskana.jobs.cleanup.task.runEvery           | period of time between the executions of the cleanup job on tasks (Duration in ISO 8601 format)                                                       | P2D                  |
```
Comment (gitgoodjhe) [inline docs/user-guide/configuration/taskana-jobs.md:52]: I'm missing the "allCompletedSameParentBusiness" property for the TaskCleanupJob
-> taskana.jobs.cleanup.task.allCompletedSameParentBusiness

and also for simplehistory:
-> taskana.jobs.cleanup.history.simple.allCompletedSameParentBusiness

File: docs/user-guide/configuration/taskana-jobs.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+| taskana.jobs.cleanup.task.firstRunAt         | first time the cleanup job on tasks is run (DateTime n ISO 8601 format)                                                              | 2021-08-03T08:00:00Z |
+| taskana.jobs.cleanup.task.minimumAge         | the completed task can be deleted by the cleanup only after this period of time or later  (Duration in ISO 8601 format)        | P10D                 |
+| taskana.jobs.cleanup.history.simple.runEvery   | period of time between the executions of the cleanup job on history events (Duration in ISO 8601 format)                                                       | P7D                  |
+| taskana.jobs.cleanup.history.simple.firstRunAt | first time the cleanup job on history events is run (DateTime n ISO 8601 format)                                                              | 2021-08-03T08:00:00Z |
+| taskana.jobs.cleanup.history.simple.minimumAge | the created history event can be deleted by the cleanup only after this period of time or later  (Duration in ISO 8601 format) | P60D                 |
```
Comment (gitgoodjhe) [inline docs/user-guide/configuration/taskana-jobs.md:57]:
```suggestion
| taskana.jobs.cleanup.history.simple.minimumAge | a history event may only be deleted by the cleanup after this period of time or later  (Duration in ISO 8601 format) | P60D                 |
```

**Review by gitgoodjhe — CHANGES_REQUESTED** [review]

File: docs/user-guide/configuration/taskana-jobs.md (hunk 2 above)
Comment (ryzheboka) [reply docs/user-guide/configuration/taskana-jobs.md:48]: What do you think about adding a "default value" column additionaly to the "sample value" column?

**Pull request merged**